src/components/FVApps/FVAppsPresentation.js

```diff
@@ -24,28 +24,43 @@ function FVAppsPresentation({ sitesWithApps }) {
             The wealth of language data uploaded by Indigenous communities onto
             their FirstVoices language sites can also be accessed through mobile
             apps. The app pulls content directly from FirstVoices and is
-            compatible with most phones, tablets, and laptops.
+            compatible with most phones, tablets, and laptops. The new apps can
+            be installed directly from the web.
           </p>
-          <p className={paraStyle}>
-            The new apps can be installed directly from the web
+          <p>
+            Install apps directly from the web by selecting “Install” or “Add to
+            Homescreen” in your browser.
+          </p>
+
+          <h2 className={headerStyle}>Installation Instructions</h2>
+          <p>
```

Comment:
I don't think we need a header here. The "Find detailed instructions..." and the link can just be tacked on to the paragraph above. So it would go "Install apps directly.... Find details instructions...: link"